locales/en-US/formautofill.properties

```diff
@@ -3,26 +3,36 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 preferenceGroupTitle = Form Autofill
-enableProfileAutofill = Enable Profile Autofill
-savedProfiles = Saved Profiles…
+enableAddressAutofill = Autofill addresses
```

Comment:
This is OK until we enable the project in Pontoon. From that moment, the [same rules](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings) as mozilla-central apply: string changes like this require new IDs.

As a general note through out the entire file: you need to add localization comments to explain the context, and use more intuitive string IDs. What is this? Where it's used? I have no idea if it's a title, a checkbox, something else.

Localizers see one string at a time and have no visibility in code or previous discussions.

Reply:
I'm going through the IDs one by one and try to make their IDs more intuitive or add comments. And yes we will make sure we change the ID if there's any string change in the future.